Classes/GlobalStateExplorers/DatabaseBrowser/FLEXSQLiteDatabaseManager.m

```diff
@@ -107,10 +107,15 @@ - (NSInteger)lastRowID {
 - (NSArray<NSString *> *)queryAllColumnsOfTable:(NSString *)tableName {
     NSString *sql = [NSString stringWithFormat:@"PRAGMA table_info('%@')",tableName];
     FLEXSQLResult *results = [self executeStatement:sql];
-
-    return [results.keyedRows flex_mapped:^id(NSDictionary *column, NSUInteger idx) {
-        return column[@"name"];
-    }] ?: @[];
+    // https://github.com/FLEXTool/FLEX/issues/554
+    if (results.keyedRows.count == 0) {
+        NSString *command = [NSString stringWithFormat:@"SELECT * FROM \"%@\" where 0=1", tableName];
```

Comment:
Can I ask why you went with this instead of the approach documented in the links you posted?

```sql
SELECT * FROM pragma_table_info('sometable')
```
https://sqlite.org/forum/info/0e55af1e0b1823f7

Reply:
For unknown reason, `SELECT * FROM pragma_table_info('sometable')` doesn't work per my testing. Since it's an ugly fallback, whatever works is the best :)

Reply:
Per my testing for some reason the `SELECT * FROM pragma_table_info('sometable')` query doesn't work. So I switched to use the other working method.

Reply:
Which iOS/Xcode version is this present in and can it be reproduced in the simulator? I want to toy around with it but I can't reproduce it